save-backend/src/main/kotlin/com/saveourtool/save/backend/service/UserDetailsService.kt

```diff
@@ -78,29 +78,54 @@ class UserDetailsService(
      * @return UserSaveStatus
      */
     @Transactional
-    fun saveUser(user: User, oldName: String?): UserSaveStatus {
-        val userName = user.name
-        return if (oldName == null) {
-            userRepository.save(user)
-            UserSaveStatus.UPDATE
-        } else if (userRepository.validateName(userName) != 0L) {
-            userRepository.deleteHighLevelName(oldName)
-            userRepository.saveHighLevelName(userName)
-            userRepository.save(user)
-            UserSaveStatus.UPDATE
-        } else {
-            UserSaveStatus.CONFLICT
+    fun saveUser(user: User, oldName: String?): UserSaveStatus = if (oldName == null) {
+        userRepository.save(user)
+        UserSaveStatus.UPDATE
+    } else if (userRepository.validateName(user.name) != 0L) {
+        userRepository.deleteHighLevelName(oldName)
+        userRepository.saveHighLevelName(user.name)
+        userRepository.save(user)
+        UserSaveStatus.UPDATE
+    } else {
+        UserSaveStatus.CONFLICT
+    }
+
+    /**
+     * @param userNameCandidate
```

Comment:
please add the description kdoc of an idea in method. 
I think only I know that why it is needed and what's for

Reply:
will do it in scope of the issue #2336